Cannot build: could not find native static library `Polly`

<!-- Thanks for the bug report! -->

### Describe the bug

Cannot build current master (90edd0c49811e8b62d24482e8d3b9c38f03bb91a), because `make build-wasmer` fails with this error:
```
error: could not find native static library `Polly`, perhaps an -L flag is missing?
error: could not compile `llvm-sys` (lib) due to 1 previous error
```

I want LLVM support, so I have followed [LLVM setup instructions](https://github.com/wasmerio/wasmer/tree/90edd0c49811e8b62d24482e8d3b9c38f03bb91a/lib/compiler-llvm#requirements).

My environment - Ubuntu 24.04 on WSL2
```
rustc 1.94.1 (e408947bf 2026-03-25)
binary: rustc
commit-hash: e408947bfd200af42db322daf0fadfe7e26d3bd1
commit-date: 2026-03-25
host: x86_64-unknown-linux-gnu
release: 1.94.1
LLVM version: 21.1.8
```

### Steps to reproduce

Assuming `docker --rm -it ubuntu:24.04`:

```
export DEBIAN_FRONTEND=noninteractive
apt update
apt install -y git curl lsb-release wget software-properties-common gnupg sudo build-essential libz-dev

wget https://apt.llvm.org/llvm.sh -O /tmp/llvm.sh
sudo bash /tmp/llvm.sh 22

curl -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain none
source "$HOME/.cargo/env"

git clone --recurse --depth=1 https://github.com/wasmerio/wasmer
cd wasmer

make build-wasmer
```

### Expected behavior

Should compile successfully.

### Actual behavior

```
# make build-wasmer
info: syncing channel updates for 1.94-x86_64-unknown-linux-gnu
info: latest update on 2026-03-26 for version 1.94.1 (e408947bf 2026-03-25)
info: downloading 6 components
-----------
INFORMATION
-----------

Host Target: `x86_64-unknown-linux-gnu`.
Enabled Compilers: cranelift, llvm, singlepass.
Testing the following compilers & engines:
  * API: cranelift llvm singlepass,
  * C-API: cranelift.
Cargo features:
  * Compilers: `--features cranelift,llvm,singlepass,wasmer-artifact-create,static-artifact-create,wasmer-artifact-load,static-artifact-load`.
Rust version: rustc 1.94.1 (e408947bf 2026-03-25).
bash: line 1: node: command not found
NodeJS version: .
LLVM version: 22.1.8.


--------------
RULE EXECUTION
--------------


cargo build  --release --manifest-path lib/cli/Cargo.toml --features cranelift,llvm,singlepass,wasmer-artifact-create,static-artifact-create,wasmer-artifact-load,static-artifact-load --bin wasmer --locked
    Updating crates.io index
...
  Downloaded block-buffer v0.12.1
  Downloaded block-buffer v0.10.4
  Downloaded bytecheck_derive v0.8.2
  Downloaded 515 crates (46.6MiB) in 6.51s (largest was `aws-lc-sys` at 9.1MiB)
  Downloaded assert_cmd v2.2.2
  Downloaded 1 crate (27.0KiB) in 0.05s
  Downloaded predicates v3.1.4
  Downloaded 1 crate (23.8KiB) in 0.05s
  Downloaded pretty_assertions v1.4.1
  Downloaded 1 crate (77.1KiB) in 0.11s
...
   Compiling lalrpop-util v0.22.2
   Compiling heck v0.3.3
   Compiling sha2 v0.10.9
error: could not find native static library `Polly`, perhaps an -L flag is missing?

error: could not compile `llvm-sys` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
make: *** [Makefile:435: build-wasmer] Error 101
```

### Additional context

I could fix this error by installing polly explicitly - `apt install -y libpolly-22-dev` and then running `make build-wasmer` again. Perhaps LLVM setup instructions should be updated.

--- a/lib/compiler-llvm/README.md
+++ b/lib/compiler-llvm/README.md
@@ -31,6 +31,7 @@ You can install LLVM easily on your Debian-like system via this command:
 ```bash
 wget https://apt.llvm.org/llvm.sh -O /tmp/llvm.sh
 sudo bash /tmp/llvm.sh 22
+sudo apt install libpolly-22-dev
 ```
 
 Or in macOS:

